### PR TITLE
Fix/accessibility improvment

### PR DESCRIPTION
**Pull request by lilia1891**
Accessibility improvements across elements (missing roles, attributes) based on Superior report.
Related Asana task and the report: https://app.asana.com/0/1205310117865974/1207079524711057/f

### CODE REVIEW

File: components/actions/ActionCard.tsx
Comment (terotik) [inline components/actions/ActionCard.tsx:1]: Should we announce this action dependency to screenreaders? @woodwoerk . Maybe we have some translation string already in place that could be used as title?

File: components/common/Card.tsx [hunk 1]
```diff
@@ -130,6 +132,8 @@ const Card = (props: CardProps) => {
             $background={imageUrl}
             $imageAlign={imageAlign}
             data-testid="image-bg"
+            role="img"
+            aria-label={t('image-credit')}
```
Comment (terotik) [inline components/common/Card.tsx:136]: Maybe this should be `image. altText` (although it's not always filled, but then the responsibility is on content editor)?

**Review by terotik — CHANGES_REQUESTED** [review]
I added some comments

File: components/actions/ActionCard.tsx
Comment (woodwoerk) [reply components/actions/ActionCard.tsx:1]: Yeah, let's do that in a separate PR, it doesn't need to block this. I'll add a task!

File: components/common/Card.tsx (hunk 1 above)
Comment (lilia1891) [reply components/common/Card.tsx]: @terotik I added a condition to use ariaText for aria-label if it's provided. Otherwise, we use generic "image"

Comment (lilia1891) [reply components/common/Card.tsx]: Also, maybe we should make the aria-text field for images required in the Admin to avoid empty aria-labels for images?

**Review by terotik — APPROVED** [review]

**Pull request merged**